Mailing.rb

```diff
@@ -0,0 +1,87 @@
+require 'net/smtp'
+require 'pp'
+require 'active_record'
+require 'time'
+require 'time_difference'
+@config = YAML.load_file('config.yml')
+
+def send_time_check
+  time = Time.now.getutc
+  puts time
+end
+
+send_time_check
+
+def mail_send
```

Comment:
Usually method name should indicate action and start with verb for example send an  "send_email"